changelog.md

```diff
@@ -8,6 +8,7 @@
 - Add new `coordinated_registrations` config option which allows for faster
   uncoordinated registrations (replaces side-effect of
   `allow_multiple_sessions`).
+- Upgraded dependency `swc`.
```

Comment:
Maybe `Upgraded dependency \`swc\`: Improves memory and cpu usage by fixing node clock update bug` to show it's fixing an actual problem?